WordPress/gutenberg
# Tag Cloud: Improve state of block with no tags

## What?

Part of https://github.com/WordPress/gutenberg/issues/63684

## Why?

Tag cloud block render message in frontend when there are no tags available to display.

## How?

Render nothing when no tags are available to display.

## Testing Instructions

- Add tag could block in post
- Select taxonomy with no terms, Save the post.
- Go to front-end, and make sure tag cloud block isn't rendered.
 
## Screenshots or screencast
